### DISCUSSION TITLE
Any way to auto hide the progress bar while a video is playing?

### DISCUSSION THREAD

**Discussion by feipoa** [smplayer-dev/smplayer#55] [Ideas]
Is there a way to auto-hide the progress bar while SMPlayer is playing a video? When viewing shorter videos, I find the active progress bar distracting.  MPV player, for example, auto hides the progress bar by default, but it also hides the play and stop buttons. Ideally, I'd like SMPlayer to auto-hide just the progress bar and keep the play buttons visible, but if all must be auto-hidden, that is okay.  I played around with the preference settings but didn't find the feature. Perhaps I overlooked something?

 Thanks.

**Comment by smplayer-dev** [comment]
The OSD bar can be hidden with the option View -> OSD -> Subtitles only.
To hide the buttons (and all the interface) you can use the compact mode, under the Video menu.

**Comment by feipoa** [comment]
Thank you for your reply, but is there a way to make the progress bar and play/pause control bar appear at the bottom of the clip with mouse-over action? Otherwise, the user has to use Ctrl-I to make the bar appear/disappear.  Thanks.

**Reply by smplayer-dev** [reply]
Preferences -> Interface -> Floating control, enable the option "show only when moving the mouse to the botton of the screen".

**Reply by feipoa** [reply]
Thank you.  Would you be willing to implement this feature in the mini GUI mode? It seems that floating control only works in Basic GUI mode, which I don't use because I find it too distracting.
